lib/codeowners/cli/owners_list_handler.rb

```diff
@@ -2,11 +2,15 @@
 
 require_relative '../checker'
 require_relative '../checker/owners_list'
+require_relative 'interactive_helpers'
 
 module Codeowners
   module Cli
     # Command Line Interface dealing with OWNERS generation and validation
     class OwnersListHandler < Base
+      include InteractiveHelpers
+
+      attr_accessor :options
```

Comment:
You can omit this line, options already included in the `Codeowners::Cli::Base` (inherited from `Thor`)

Reply:
thanks!